### DISCUSSION TITLE
Can't access from mobile browser to xfce repo

### DISCUSSION THREAD

**Discussion by GJoe2** [Yisus7u7/termux-desktop-xfce#69] [Q&A] [answered]
I was looking the xfce script, and trying to figure  out if adding xfce repo would have some issues with actual official termux repo or extras repos, but I am not sure how to navigate and see what packages your private repo have

This is what I see inside your source list

deb [trusted=yes arch=all] https://Yisus7u7.github.io/termux-desktop-xfce/packages desktop-xfce extras 

I tried to copy and paste this link:

https://Yisus7u7.github.io/termux-desktop-xfce/packages

But I got an error (error 404)

**Comment by JesusChapman** [comment, marked as answer]
https://github.com/Yisus7u7/termux-desktop-xfce/tree/gh-pages

There are the packages
